### ISSUE TITLE
REST /ping doesn't check Redis storage status

### ISSUE THREAD

**Issue by haron** [Netflix/dynomite#567]
http://localhost:22222/ping returns OK regardless of Redis status (e.g. when all Redis nodes are stopped). This behavior contradicts the [documentation](https://github.com/Netflix/dynomite/wiki/REST):

> ping: performs a Redis inline ping to test Dynomite and storage

**Comment by ipapapa** [comment]
You are correct. There is a bug in the response of the REST call. The `PING` in Dynomite works fine some there is a small disconnect there. We are not using that end-point. Let me know if you want to fix it or we can take care of it.
